src/dotnet/HQ.Server/Services/ProjectServiceV1.cs

```diff
@@ -162,13 +168,68 @@ public async Task<Result<GetProjectsV1.Response>> GetProjectsV1(GetProjectsV1.Re
             QuoteId = t.QuoteId,
             QuoteNumber = t.Quote != null ? t.Quote.QuoteNumber : null,
             HourlyRate = t.HourlyRate,
-            BookingHours = t.BookingHours,
             BookingPeriod = t.BookingPeriod,
             StartDate = t.StartDate,
             EndDate = t.EndDate,
             BillingEmail = t.Client.BillingEmail,
             OfficialName = t.Client.OfficialName,
-            Status = t.Quote != null ? (int)t.Quote.Status : 0
+            Status = t.Quote != null ? (int)t.Quote.Status : 0,
+            ProjectStatus = t.Status,
+
+
+            BookingStartDate = t.ChargeCode!.Times.Where(x => x.Date >= bookingStartDate && x.Date <= bookingEndDate && x.Date <= bookingEndDate).Min(x => x.Date),
+            BookingEndDate = t.ChargeCode!.Times.Where(x => x.Date >= bookingStartDate && x.Date <= bookingEndDate && x.Date <= bookingEndDate).Max(x => x.Date),
+            BookingHours = t.ChargeCode!.Times.Where(x => x.Date >= bookingStartDate && x.Date <= bookingEndDate && x.Date <= bookingEndDate).Sum(x => x.Hours),
+            BookingAvailableHours = t.BookingHours - t.ChargeCode!.Times.Where(x => x.Date >= bookingStartDate && x.Date <= bookingEndDate && x.Date <= bookingEndDate).Sum(x => x.Hours),
+            BookingPercentComplete = t.BookingHours == 0 ? 0 : t.ChargeCode!.Times.Where(x => x.Date >= bookingStartDate && x.Date <= bookingEndDate && x.Date <= bookingEndDate).Sum(x => x.Hours) / t.BookingHours,
+
+            TotalHours = t.ChargeCode!.Times.Where(x => x.Date <= t.EndDate).Sum(x => x.Hours),
+            TotalAvailableHours = t.TotalHours != null ? t.TotalHours.Value - t.ChargeCode!.Times.Where(x => x.Date <= bookingEndDate).Sum(x => x.Hours) : null,
+            TotalPercentComplete = !t.TotalHours.HasValue || t.TotalHours == 0 ? null : t.ChargeCode!.Times.Where(x => x.Date <= bookingEndDate).Sum(x => x.Hours) / t.TotalHours.Value,
+            TotalPercentCompleteSort = !t.TotalHours.HasValue || t.TotalHours == 0 ? -1 : t.ChargeCode!.Times.Where(x => x.Date <= bookingEndDate).Sum(x => x.Hours) / t.TotalHours.Value,
+            TotalStartDate = t.ChargeCode.Times.Where(x => x.Date <= bookingEndDate).Min(t => t.Date),
+            TotalEndDate = t.ChargeCode.Times.Where(x => x.Date <= bookingEndDate).Max(t => t.Date),
```

Comment:
We can remove the `x.Date <=` condition for total calculations, we are interested in all time so no need to limit it.